Content.Shared/SS220/CriminalRecords/CriminalRecordsConsoleComponent.cs

```diff
@@ -140,3 +140,17 @@ public DeleteCriminalRecordStatus(int time)
         Time = time;
     }
 }
+
+[Serializable, NetSerializable]
+public sealed class UpdateCriminalRecordStatusEvent(
```

Comment:
```suggestion
[ByRefEvent]
public record struct UpdateCriminalRecordStatusEvent(
```
Сам же показывал мне гайд с правильным оформлением ивентов!

Reply:
> Сам же показывал мне гайд с правильным оформлением ивентов!

С RaisePredictiveEvent такая тема не канает

Reply:
=(